tests/plan_parser_test.py

```diff
@@ -272,6 +272,486 @@ def test_order_by(self) -> None:
         self._expect_rdd(node=rdd, expected_schema=schema)
 
 
+    """
+    From here, all tests are only considering one transformation
+    """
+
+    def test_zz_aggregate(self) -> None:
```

Comment:
zz?

Reply:
Sorry, I forgot removing it.
It was because pytest sorts the tests alphabetically, and the column id of the conditions changed every time I added a new test.
Apart of that, I renamed the test suite